jekyll/_cci2/workflows.md

```diff
@@ -430,6 +430,48 @@ workflows:
               ignore: /.*/
 ```
 
+In the example below, two jobs are defined (`test` and `deploy`) and three workflows utilize those jobs:
+
+- The `build` workflow runs for all branches except `main` and is not run on tags.
```

Comment:
For this to be true do we need to add the `tags:` `ignore:` keys for the `build` workflow?

Reply:
@rosieyohannan great question! You actually don't since we don't automatically run workflows on tags, so by not including it doesn't run on tags. I set up this config example in one of my test accounts, and you can see the three scenarios here:

![Screen Shot 2020-10-15 at 8 06 26 AM](https://user-images.githubusercontent.com/19393321/96130408-8224f500-0ebd-11eb-9baa-305ea8e9af93.png)

Reply:
Ah interesting! Thanks :-)

Reply:
Maybe we can add a comments to say this explicitly if it's not too wordy?